relay-event-normalization/src/normalize/span/description/sql/mod.rs

```diff
@@ -130,9 +130,9 @@ fn scrub_queries_inner(db_system: Option<&str>, string: &str) -> (Option<String>
     let mut string = Cow::from(string.trim());
 
     for (regex, replacement) in [
-        (&NORMALIZER_REGEX, "$pre%s"),
         (&COMMENTS, "\n"),
         (&INLINE_COMMENTS, ""),
+        (&NORMALIZER_REGEX, "$pre%s"),
```

Comment:
Comments may contain string delimiters (`'`), and strings may contain comment delimiters. So no matter how we order the regexes, one can construct a case where scrubbing does not work. Prefer this order for now because we've seen that the previous order destroyed active record descriptions.